torch/csrc/jit/passes/quantization/helper.cpp

```diff
@@ -133,6 +139,31 @@ std::vector<std::string> _single_input_general_value_aten_funcs = {
     "leaky_relu_",
 };
 
+std::tuple<c10::QScheme, QParamVector> _per_tensor_asym_qparam =
```

Comment:
can we add some comments on where the numerical values are coming from?  Even if we can't reuse the constants directly, some context for future code readers on where they are coming from would be helpful